src/views/splash/presentation.jsx

```diff
@@ -581,7 +556,6 @@ SplashPresentation.propTypes = {
     onHideEmailConfirmationModal: PropTypes.func.isRequired,
```

Comment:
Oh thanks for catching this! There was some left-over logic related to closing and opening the pre-existing iframe modal that we don't need anymore because we moved that logic into the banner and modal themselves. Removed that prop (and some others) in my most recent commit.